components/date-picker/createPicker.tsx

```diff
@@ -135,7 +135,7 @@ export default function createPicker(TheCalendar): any {
       ) : null;
 
       const input = ({ value: inputValue }) => (
-        <div>
+        <div className={props.className}>
```

Comment:
`props.className` should be here?

https://github.com/EnrikoLabriko/ant-design/blob/1f79c0b1760b616e917435677a002b6a689dad37/components/date-picker/createPicker.tsx#L157